relational_database/homework.py

```diff
@@ -0,0 +1,234 @@
+from typing import List
+
+
+def task_1_add_new_record_to_db(con) -> None:
+    """
+    Add a record for a new customer from Singapore
+    {
+        'customer_name': 'Thomas',
+        'contactname': 'David',
+        'address': 'Some Address',
+        'city': 'London',
+        'postalcode': '774',
+        'country': 'Singapore',
+    }
+
+    Args:
+        con: psycopg connection
+
+    Returns: 92 records
+    """
+    with con.cursor() as cursor:
+        cursor.execute("INSERT INTO customers VALUES('92', 'Thomas', 'David',"
+                       " 'Some Address', 'London','774', 'Singapore')");
```

Comment:
тут і у всіх наступних завданнях: тобі не потрібно крапки з комою після кожного sql запиту. Я так розумію, крапка з комою мали бути всередині запиту

Також, мб буде цікаво глянути тему про крапку з комою в пайтоні:
```suggestion
                       " 'Some Address', 'London','774', 'Singapore');")
```
https://stackoverflow.com/a/8236402/8995868